api/controllers/install/controller.go

```diff
@@ -49,9 +50,14 @@ type InstallController struct {
 	airgapBundle         string
 	configValues         string
 	endUserConfig        *ecv1beta1.Config
+	envSetter            EnvSetter
 	mu                   sync.RWMutex
 }
 
+type EnvSetter interface {
+	Setenv(key string, val string) error
+}
```

Comment:
Wondering if it would make sense for this interface, the respective struct implementation and the mock implementation to live in `api/pkg/utils` instead? 🤔

Reply:
moved to the runtimeconfig package